libcodechecker/libhandlers/cmd.py

```diff
@@ -353,7 +373,227 @@ def __register_suppress(parser):
+                                 "the analysis results for this product.")
+
+        parser.add_argument('-n', '--name',
+                            type=str,
+                            dest="display_name",
+                            default=argparse.SUPPRESS,
+                            required=False,
+                            help="A custom display name for the product, "
+                                 "which will be shown in the viewer. This "
+                                 "is purely for decoration and user "
+                                 "experience, program calls use the "
+                                 "PRODUCT_NAME.")
+
+        parser.add_argument('--description',
+                            type=str,
+                            dest="description",
+                            default=argparse.SUPPRESS,
+                            required=False,
+                            help="A custom textual description to be shown "
+                                 "alongside the product.")
+
+        dbmodes = parser.add_argument_group(
+            "database arguments",
+            "NOTE: These database arguments are relative to the server "
+            "machine, as it is the server which will make the database "
+            "connection.")
+
+        dbmodes = dbmodes.add_mutually_exclusive_group(required=False)
+
+        SQLITE_PRODUCT_NAME_DEFAULT_VAR = '<PRODUCT_NAME>.sqlite'
+        dbmodes.add_argument('--sqlite',
+                             type=str,
+                             dest="sqlite",
+                             metavar='SQLITE_FILE',
+                             default=SQLITE_PRODUCT_NAME_DEFAULT_VAR,
+                             required=False,
+                             help="Path of the SQLite database file to use. "
+                                  "Not absolute paths will be relative to "
+                                  "the server's <CONFIG_DIRECTORY>.")
+
+        dbmodes.add_argument('--postgresql',
+                             dest="postgresql",
+                             action='store_true',
+                             required=False,
+                             default=argparse.SUPPRESS,
+                             help="Specifies that a PostgreSQL database is "
+                                  "to be used instead of SQLite. See the "
+                                  "\"PostgreSQL arguments\" section on how "
+                                  "to configure the database connection.")
+
+        PGSQL_PRODUCT_NAME_DEFAULT_VAR = '<PRODUCT_NAME>'
+        pgsql = parser.add_argument_group(
+            "PostgreSQL arguments",
+            "Values of these arguments are ignored, unless '--postgresql' is "
+            "specified! The database specified here must exist, and be "
+            "connectible by the server.")
+
+        # TODO: --dbSOMETHING arguments are kept to not break interface from
```

Comment:
These TODOs will be cleared later on, as part of #745. Right now, this is here, to keep the argument list akin to that of `server`.